libs/search-widget/src/widgets/row/Row.svelte

```diff
@@ -65,19 +65,25 @@
   .row {
     cursor: pointer;
     padding: 16px;
-    margin-bottom: 50px;
+    margin-bottom: 40px;
     background-color: var(--color-light-stronger);
   }
-  .row:focus,
-  .row:hover {
+  .row:focus-visible {
     border-color: var(--color-dark-stronger);
     padding-left: 13px;
     border-left: 3px solid var(--color-neutral-strong);
     outline: 0px;
   }
   .title {
+    margin-bottom: 0.4em;
     font-weight: var(--font-weight-bold);
   }
+  .byline {
+    font-size: 12px;
+    font-style: italic;
+    padding-bottom: 10px;
+    border-bottom: 1px solid #000;
```

Comment:
use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

Reply:
fixed